style.css

```diff
@@ -111,3 +121,281 @@ body {
   transition: 1s ease-in-out;
   cursor: pointer;
 }
+
+.cards {
+  display: grid;
+  gap: 88px;
+  background-color: #e5e5e5;
+  margin-top: 100px;
+}
+
+.card {
+  width: 311px;
+  height: 552px;
+  border: 2px solid lightgray;
+  border-radius: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  position: relative;
+  left: -15px;
+  background-color: #fff;
+}
+
+.card-img {
+  width: 311px;
+  height: 220px;
+  border-radius: 8px;
+  margin-top: 11px;
+  margin-left: 14px;
+}
+
+.connect-2 {
+  color: #7f8cff;
+  font-size: 16px;
+  padding-top: 12px;
+  padding-bottom: 12px;
+}
+
+.card-name h2 {
+  width: 295px;
+  height: 36px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.card-props {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  height: 32px;
+  position: relative;
+  bottom: 39px;
+  padding-left: 5px;
+  left: -26px;
+}
+
+.card-prop a {
+  position: relative;
+  right: 8px;
+  height: 16px;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  color: #7a869a;
+  margin-left: 5px;
+  text-decoration: none;
+}
+
+.card-props li:first-child a {
+  font-weight: 600;
+  text-transform: uppercase;
+  color: #2e4662;
+}
+
+.card-text {
+  position: relative;
+  bottom: 73px;
+  width: 295px;
+  height: 76px;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.card-languages {
+  display: flex;
+  position: relative;
+  right: 67px;
+  bottom: 103px;
+  justify-content: space-between;
+}
+
+.card-language {
+  font-size: 12px;
+  background-color: #eaebfe;
+  padding: 4px 12px;
+  border-radius: 8px;
+  color: #6070ff;
+  margin-left: 8px;
+}
+
+.card-btn {
+  font-size: 19px;
+  text-align: start;
+  padding: 12px 12px;
+  border: 1px solid #6070ff;
+  width: 131px;
+  border-radius: 8px;
+  margin-bottom: 15px;
+  color: #396df2;
+  background: white;
+  cursor: pointer;
+  position: relative;
+  bottom: 114px;
+  height: 49px;
+  right: 28%;
+}
+
+.about-section {
+  position: relative;
+  left: -30.8px;
+  border-top-right-radius: 98px;
+  margin-top: 185px;
+  height: 984px;
+  width: 373px;
+}
+
+.about-heading {
+  width: 327px;
+  height: 35px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.about-text {
+  width: 327px;
+  height: 147px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.resume-btn {
+  font-size: 19px;
+  text-align: start;
+  padding: 12px 12px;
+  border: 1px solid #6070ff;
+  width: 153px;
+  border-radius: 8px;
+  margin-bottom: 15px;
+  color: #396df2;
+  background: white;
+  cursor: pointer;
+}
+
+.languages,
+.frameworks,
+.skills {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  right: 34px;
+}
```

Comment:
- [ ] Kindly consider reducing the spacing around your content as currently, the spacing around it is too much and also doesn't match that of the `Figma` design given. This will help make your website look more appealing and create a good user experience. See below for more details:
<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="374" alt="Screenshot 2023-06-06 at 20 22 31" src="https://github.com/josman227/Mobile-Portfolio/assets/107192076/5255484d-2b23-4531-a0a2-cdf2115a6d37">
</td>
<td>
<img width="368" alt="Screenshot 2023-06-06 at 20 21 45" src="https://github.com/josman227/Mobile-Portfolio/assets/107192076/cf92c6c8-8721-4aaf-b44e-3d862e3104d3">
</td>
</tr>
</table>